src/bttrx_wifi.cpp

```diff
@@ -164,9 +166,21 @@ void BTTRX_WIFI::setup(BTTRX_CONTROL *control)
 	}
 	bttrx_control_ = control;
 
+	Serial.println("");
 	Serial.println("Configuring access point...");
 
-	WiFi.softAP(WIFI_SSID, WIFI_PASSWORD);
+	String callsign = preferences.getString("callsign", "");
+	if (callsign != "") {
+		char result[sizeof(WIFI_SSID)+7];
+		strcpy(result, WIFI_SSID);
+		strcat(result, "_");
+		strcat(result, callsign.c_str());
+		Serial.print("WiFi SSID: ");
+		Serial.println(result);
+		WiFi.softAP(result, WIFI_PASSWORD);
+	} else {
+		WiFi.softAP(WIFI_SSID, WIFI_PASSWORD);
+	}
```

Comment:
Please try to get rid of this if/else. Build your SSID string, then call WiFi.softAP(...). SSID building should happen in a separate method for better testability (e.g. `buildSSID(string prefix, string suffix)`, called i.e. `buildSSID(WIFI_SSID, bttrx_control.get(kCallsign));`)